index.html

```diff
@@ -109,40 +150,8 @@ <h3>Other dashboards</h3>
-                        <div class="dashboard-description">Please <a href="https://github.com/mozilla/telemetry-dashboard/issues">Report a Bug</a> if you require this data.</div>
-                    </span>
-                    <span class="list-group-item disabled">
-                        Addon Shutdown Correlations
-                        <div class="dashboard-description">Please <a href="https://github.com/mozilla/telemetry-dashboard/issues">Report a Bug</a> if you require this data.</div>
-                    </span>
-                    <span class="list-group-item disabled">
-                        Chrome Hangs
-                        <div class="dashboard-description">Please <a href="https://github.com/mozilla/telemetry-dashboard/issues">Report a Bug</a> if you require this data.</div>
-                    </span>
-                    <span class="list-group-item disabled">
-                        Fennec App Not Responding (ANR)
-                        <div class="dashboard-description">Please <a href="https://github.com/mozilla/telemetry-dashboard/issues">Report a Bug</a> if you require this data.</div>
-                    </span>
-                    <span class="list-group-item disabled">
-                        Fennec UI Telemetry
-                        <div class="dashboard-description">Please <a href="https://github.com/mozilla/telemetry-dashboard/issues">Report a Bug</a> if you require this data.</div>
-                    </span>
-                    <span class="list-group-item disabled">
-                        Firefox UI Telemetry
-                        <div class="dashboard-description">Please <a href="https://github.com/mozilla/telemetry-dashboard/issues">Report a Bug</a> if you require this data.</div>
-                    </span>
-                    <span class="list-group-item disabled">
-                        Main Thread I/O
-                        <div class="dashboard-description">Please <a href="https://github.com/mozilla/telemetry-dashboard/issues">Report a Bug</a> if you require this data.</div>
-                    </span>
-                    <span class="list-group-item disabled">
-                        Power Dashboard <span class="label label-warning" style="font-size: 12px">DRAFT</span>
-                        <div class="dashboard-description">Please <a href="https://github.com/mozilla/telemetry-dashboard/issues">Report a Bug</a> if you require this data.</div>
-                    </span>
-                    <span class="list-group-item disabled">
-                        SlowSQL
-                        <div class="dashboard-description">Please <a href="https://github.com/mozilla/telemetry-dashboard/issues">Report a Bug</a> if you require this data.</div>
+                        Missing something?
+                        <div class="dashboard-description">Please <a href="https://github.com/mozilla/telemetry-dashboard/issues">Report a Bug</a> if you are missing a dashboard here.</div>
```

Comment:
You've got 2 Report Bug links that point to the same place. Is the one up top necessary? I generally *do* look at the bottom for bug-report links.

Reply:
I thought it might be a friendly "next step" for people looking through "other tools" and not finding what they found.
On the other hand, the top-right link covers looking around generally when something is broken. Does the "getting help" section cover this now?

Reply:
I'm dropping this one, it's probably not adding anything additionally and potentially hidden behind scroll.